src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.hpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+
+#include "llm_compiled_model.hpp"
+#include "openvino/core/descriptor/output.hpp"
+#include "openvino/runtime/isync_infer_request.hpp"
+
+namespace ov {
+namespace npuw {
+
+class LLMInferRequest final : public ov::ISyncInferRequest {
+public:
+    explicit LLMInferRequest(const std::shared_ptr<ov::npuw::LLMCompiledModel>& compiled_model,
+                             const ov::npuw::LLMCompiledModel::KVCacheDesc& kvcache_desc);
+
+    void infer() override;
+
+    ov::SoPtr<ov::ITensor> get_tensor(const ov::Output<const ov::Node>& port) const override;
+
+    void check_tensors() const override{};
+
+    virtual std::vector<ov::ProfilingInfo> get_profiling_info() const {
+        return {};
+    }
+    virtual std::vector<ov::SoPtr<ov::IVariableState>> query_state() const {
```

Comment:
`/__w/openvino/openvino/openvino/src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.hpp:30:56: error: 'virtual std::vector<ov::SoPtr<ov::IVariableState> > ov::npuw::LLMInferRequest::query_state() const' can be marked override [-Werror=suggest-override]
   30 |     virtual std::vector<ov::SoPtr<ov::IVariableState>> query_state() const {`

Reply:
Fixed, thanks!